tool/amdllpc.cpp

```diff
@@ -999,6 +1000,10 @@ static Result BuildPipeline(
 #if LLPC_CLIENT_INTERFACE_MAJOR_VERSION >= 21
             pShaderInfo->entryStage = pCompileInfo->shaderModuleDatas[i].shaderStage;
 #endif
+#if LLPC_CLIENT_INTERFACE_MAJOR_VERSION >= 35
+            pShaderInfo->options.disableLicm |= llvm::cl::DisableLicm;
```

Comment:
please merge llvm::cl::DisableLicm and options.disableLicm in LLPC source  code, instead of AMDLLPC.
we often use llpcOptions in vk panel to adjust the tuning options for debugging, so we want to keep global options work in both amdllpc and vk driver. thanks!